### PR TITLE
FAQ documentation bullet list

### PR DESCRIPTION
**Pull request by hvy**
Adds a bullet list at the top of the FAQ section in the documentation for quick overview. This bullet list is easier to go through than the list on the sidebar as long sentences on the sidebar are wrapped.

Screen Shot 2020-01-23 at 12 37 17

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
 FAQ
 ===
 
+- :ref:`Can I use Optuna with X? (where X is your favorite ML library)`
```
Comment (sile) [inline docs/source/faq.rst]: I'm not 100% sure, but we might be able to use `..contents::` directive to generate the same HTML with fewer maintenance cost.
```
.. contents::
   :local:
```

Comment (hvy) [reply docs/source/faq.rst]: That definitely seems like the right way to go. Thanks, applied!

**Comment by hvy** [comment]
PTAL.

File: docs/source/faq.rst (hunk 1 above)
Comment (hvy) [reply docs/source/faq.rst]: Just leaving a comment about the directive. https://stackoverflow.com/questions/24129481/how-to-include-a-local-table-of-contents-into-sphinx-doc

**Pull request merged**